package.json

```diff
@@ -26,95 +26,95 @@
     "@aeternity/bip39": "^0.1.0",
     "@aeternity/hd-wallet": "^0.2.0",
     "@aeternity/ledger-app-api": "^0.1.0",
-    "@ledgerhq/hw-transport-u2f": "^4.72.1",
+    "@ledgerhq/hw-transport-u2f": "^4.78.0",
     "@stamp/it": "^1.1.0",
     "@vue/cli-plugin-e2e-cypress": "^3.12.0",
     "@vue/cli-service": "^3.12.0",
     "@zxing/library": "^0.15.2",
     "aeternity-tokens": "^1.0.2",
-    "argon2-browser": "^1.5.2",
-    "axios": "^0.19.0",
+    "argon2-browser": "^1.13.0",
```

Comment:
Last time there was a problem with argon2 update, does this work as intended or is breaking the build. 

Also are we still using argon2 somewhere after the switch to web crypto #219 ?